cmd/srcd/cmd/sql_test.go

```diff
@@ -1,28 +1,218 @@
-// +build integration
-
 package cmd
 
 import (
 	"bytes"
 	"context"
+	"fmt"
 	"io"
 	"io/ioutil"
 	"log"
 	"os"
 	"os/exec"
 	"path"
 	"path/filepath"
+	"regexp"
 	"strings"
 	"testing"
 	"time"
 
+	"github.com/kr/pty"
```

Comment:
Will this work on windows?

Reply:
Ouch. I guess not, by looking [here](https://github.com/kr/pty/issues/50) it seems that there's a possibility to fallback to other methods, but I haven't explored it yet. Maybe we could skip this test if `runtime.GOOS == "windows"` for now and open an issue.

Reply:
Please take a look into it, if possible we should support windows too in the tests.

Reply:
but let's do it in a separate PR. 
I mean it makes sense to disable this test on windows right now but the proper fix (if such fix is possible) better implement in a separate pr.

Reply:
ok but let's make sure the test is properly skipped in windows, and open a new issue to keep track of it.

Reply:
Here it is #411.